roles/cobbler/templates/snippets/cephlab_packages_rhel

```diff
@@ -1,3 +1,8 @@
 ## {{ ansible_managed }}
+#set os_version = $getVar('os_version','')
```

Comment:
It's just a part of the cobbler snippet API; since it's in a cobbler snippet I think it's reasonable to think a user might do something like google "cobbler os_version"